src/main/scala/format/pgn/model.scala

```diff
@@ -18,15 +18,13 @@ object PgnStr extends OpaqueString[PgnStr]
 
 case class Pgn(tags: Tags, turns: List[Turn], initial: Initial = Initial.empty):
 
-  // index is NOT a full move turn!
-  private def updateTurnAt(index: Int, f: Turn => Turn) =
-    // val index = fullMove.value - 1
-    (turns lift index).fold(this) { turn =>
-      copy(turns = turns.updated(index, f(turn)))
+  private def updateTurnAt(ply: Ply, f: Turn => Turn) =
+    (turns lift ply.value).fold(this) { turn =>
```

Comment:
`index` is not a ply (and not a full move turn either) so I don't think `turns lift ply.value` will work